static/js/debug.js

```diff
@@ -131,3 +131,32 @@ IterationProfiler.prototype = {
         }
     },
 };
+
+export function test_send_many_messages(stream, subject, count, msg) {
```

Comment:
I have manually tested this function in chrome's console using 
`debug.test_send_many_messages('Denmark', 'Verona3', 10)`. Messages are sent but I faced a problem which I couldn't get around. What happens is that whenever this function sends messages, there are two messages rendered on front end. Its only after a browser refresh that the extra messages rendered disappear.